src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -116,16 +113,13 @@ public static void sendBuildCompleteNotification(Run run, TaskListener listener)
 
         for (Webhook webhook : property.getWebhooks()) {
             if (shouldSendNotification(webhook, listener, run)) {
-                //            listener.getLogger().println(String.format("Notifying webhook '%s'", webhook));
                 try {
                     HttpWorker worker = new HttpWorker(run.getEnvironment(listener).expand(webhook.getUrl()), gson.toJson(card), webhook.getTimeout(), 3, listener.getLogger());
                     executorService.submit(worker);
-                } catch (Throwable error) {
-                    error.printStackTrace(listener.error(String.format("Failed to notify webhook '%s'", webhook)));
-                    listener.getLogger().println(String.format("Failed to notify webhook '%s' - %s: %s", webhook, error.getClass().getName(), error.getMessage()));
+                } catch (IOException | InterruptedException | RejectedExecutionException e) {
```

Comment:
block of code that submits the notification is repeated - will move it to separate class later